arangod/Cluster/ClusterMethods.cpp

```diff
@@ -1889,16 +1891,23 @@ Result fetchEdgesFromEngines(transaction::Methods& trx,
 
     VPackSlice edges = resSlice.get("edges");
     bool allCached = true;
+    VPackArrayIterator allEdges(edges);
+    result.reserve(allEdges.size());
 
-    for (VPackSlice e : VPackArrayIterator(edges)) {
+    for (VPackSlice e : allEdges) {
       VPackSlice id = e.get(StaticStrings::IdString);
       if (!id.isString()) {
         // invalid id type
         LOG_TOPIC("a23b5", ERR, Logger::GRAPHS)
             << "got invalid edge id type: " << id.typeName();
         continue;
+      
+      }
+      if (result.capacity() == 0) {
+        result.reserve(16);
```

Comment:
Just a question: Why 16 here?

Reply:
Let me rephrase:
Why at all?

there is a reserve on the exact number of results a couple of lines above, so this of no use.
I will remove

Reply:
(I think the initial intention was to avoid to small rescaling steps and 16 was an arbitrary number)